### ISSUE TITLE
Review comments

### ISSUE THREAD

**Issue by ellisonbg** [ipython/sloan-2013-reports#4]
I don't want to submit a PR as it would conflict with other changes being made. Notebooks are still painful for this type of thing!

Comments by section:
# Activities

"bit longer to solve than . " missing "expected"
# Developer meetings
- We did have a smaller meeting last Jan that was funded by Sloan.
- "interactive Javascript framework" -> "interactive JavaScript widget architecture"
- Include the year of each dev meeting (it is not quite clear).
# Talks, seminars and other outreach

The formatting is not consistent between items. Sometimes, the type of talk ("Keynote", etc.) is immediately after the speakers, sometimes after the title, sometimes missing.

I think the description of our weekly lab meetings on Google+ should be in the previous "Developer meetings" sections as that is really what they are.
# Papers and other publications

I would clarify that these paper are ones that we ourselves have been involved in. Otherwise it is not quite clear how these differ from the papers listed below. Especially confusing as one of our paper is listed in both places.
# NBViewer as a sharing platform

Hmm, I thought that make on heroku we were getting around 30-40k hits per week, not per month. I guess the analytics check will confirm that.
# Courses using IPython as a central element

Also see

Josh Attenberg's Practical Data Science at NYU:

http://jattenberg.github.io/PDS-Fall-2013/

Jennifer Klays, Computational Physics at Cal Poly (no GitHub repo)
# Commercial projects

Also, Sense:

https://senseplatform.com/

**Comment by Carreau** [comment]
Also might want to add: 

https://github.com/mattpap/IAldor

Which mainly in C, might be usefull for other languages that already have a C-library.

>  30-40k hits per week

the factor of 2 is the differences between Visits/ And pageview.

in roughly 7 month March-Sept

Visits 507,540

Unique Visitors 248,479

Pageviews 941,507

Pages / Visit 1.86

Avg. Visit Duration 00:02:30

Bounce Rate 68.94%

New Visits 48.96%

**Comment by fperez** [comment]
Great, thanks!!

I fixed everything _except_ for the talks formatting: I knew it wasn't perfectly consistent when I did it b/c I was pasting from multiple sources, but I honestly ran out of patience to do that much manual editing.  We should just put all that into a database so we can generate consistently formatted output...

For now I'm going to leave that as-is, I'm sure we'll live :)

Otherwise I fixed everything else you guys noted.  Thanks for all the review, sending it now to Josh!
